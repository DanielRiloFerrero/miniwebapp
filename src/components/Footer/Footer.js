import React from 'react'

import { Container } from 'reactstrap'

import styles from './Footer.module.scss'

const Footer = () => {
  return (
    <footer className={styles.footer}>
      <Container>
        Desarrollado por{' '}<a href='https://www.linkedin.com/in/danielrf/'>Daniel Rilo</a>
      </Container>
    </footer >
  )
}

export default Footer