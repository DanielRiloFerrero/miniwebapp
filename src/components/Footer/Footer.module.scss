@import '../../_assets/styles/abstracts/vars.scss';

.footer {
  flex-shrink: 0;
  background-color: $primary-color;
  padding: $p-xs 0;
  color: $secondary-color;
}