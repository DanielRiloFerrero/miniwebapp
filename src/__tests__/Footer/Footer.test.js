import React from "react"
import { render, cleanup } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import Footer from "../../components/Footer/Footer"

describe('Footer tests', () => {
  it("render", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot()
  })
})