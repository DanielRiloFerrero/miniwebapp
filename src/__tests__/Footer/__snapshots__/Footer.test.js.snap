// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests render 1`] = `
<div>
  <footer
    class="footer"
  >
    <div
      class="container"
    >
      Desarrollado por
       
      <a
        href="https://www.linkedin.com/in/danielrf/"
      >
        Daniel Rilo
      </a>
    </div>
  </footer>
</div>
`;
